decko-rails/lib/decko/rails.rb

```diff
@@ -1,11 +1,33 @@
 DECKO_RAILS_GEM_ROOT = File.expand_path('../../..', __FILE__)
 
+require 'rails/all'
+require 'decko/engine'
+
 module Decko
-  module Rails
+  module Rails # not sure we need this
     class << self
       def gem_root
         DECKO_RAILS_GEM_ROOT
       end
     end
   end
-end
\ No newline at end of file
+
+  if defined? ::Rails::Railtie
```

Comment:
Actually, the reason for it landing here even if there's nothing in Decko::Rails is that this file is the standard require for the name.  We don't have to even define that constant, but since this is the included file I thought it fine to put Railtie stuff here.